hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/ReconConstants.java

```diff
@@ -46,6 +46,8 @@ public final class ReconConstants {
   public static final String RECON_QUERY_BATCH_PARAM = "batchNum";
   public static final String RECON_QUERY_PREVKEY = "prevKey";
   public static final String RECON_QUERY_START_PREFIX = "startPrefix";
+  public static final String RECON_QUERY_MAX_CONTAINER_ID = "maxContainerId";
+  public static final String RECON_QUERY_MIN_CONTAINER_ID = "minContainerId";
```

Comment:
These same changes appear in https://github.com/apache/ozone/pull/8796 , may need rebase of code ?